k8s/nginx/staging/netmanager-vs.yaml

```
@@ -63,7 +63,9 @@ spec:
   routes:
     - path: /
       action:
-        pass: platform-ui
+        redirect:
+          url: https://staging-vertex.airqo.net
+          code: 301      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)